docs/help/_posts/2010-05-10-configure.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Safe configuration and overlapping calls
+layout: post
+---
+
+Sometimes we want to configure a call that overlaps a more general call we have previously setup to run a [callback](/help/callbacks) or throw an [exception](/help/throwing-exceptions/). Ideally we would modify the setups so they don't overlap, but we can also prevent these callbacks from running while we setup the next call by calling `Configure()` before performing additional `Returns()`. `Configure()` tells NSubstitute we are configuring a call so that it will not run any callbacks from previous configurations.
+
+{% requiredcode %}
+public interface ICalculator { int Add(int a, int b); }
+ICalculator calculator;
+[SetUp] public void SetUp() { calculator = Substitute.For<ICalculator>(); }
+{% endrequiredcode %}
+
+{% examplecode csharp %}
+calculator.Add(Arg.Any<int>(), Arg.Any<int>()).Returns(x => { throw new Exception(); });
+
+// Now if we try to add a return value for a more specific returns this will throw
+// before it gets a chance to configure the call:
+//    calculator.Add(1, 2).Returns(3);
+
+// Instead, we can use Configure to ensure the previous callback does not run:
+calculator.Configure().Add(1, 2).Returns(3);
+
+// Now both the exception callback and our other return have been configured:
+Assert.That(3, calculator.Add(1, 2));
+Assert.Throws<Exception>(() => calculator.Add(-2, -2));
+{% endexamplecode %}
+
+NSubstitute will also assume we are configuring a call if we have an argument matcher in our call, such as `calculator.Add(Arg.Is(1), 2).Returns(3)`, but it is generally better to me more explicit by using `.Configure()`.
```

Comment:
Probably it's better to clarify what is argument matcher. For instance:
"such as `Arg.Is(1)` in `calculator.Add(Arg.Is(1), 2).Returns(3)`"